code/game/objects/items/wizard_weapons.dm

```diff
@@ -77,12 +77,12 @@
 	return .
 
 /obj/item/mjollnir
-	name = "Mjolnir"
+	name = "Mjollnir"
```

Comment:
```suggestion
	name = "Mjöllnir"
```
If you want to accentuate it for authenticity or not. It's not mandatory, I'll merge the PR anyway.

Reply:
No accentuated o.